nemo_text_processing/text_normalization/ja/data/whitelist.tsv

```diff
@@ -0,0 +1,40 @@
+Dr.	ドクター
+dr.	ドクター
```

Comment:
Here I may handle capitalization as part of the grammar. Again, doing so helps to reduce the size of the graph.